src/library_wasmfs.js

```diff
@@ -25,6 +25,18 @@ FS.createPreloadedFile = FS_createPreloadedFile;
 #endif
   ],
   $FS : {
+    ErrnoError: null,
```

Comment:
Maybe `WasmFSError` and not `ErrnoError`? I think that's more general, and we may want the error to cover things not possible in `errno` values in theory.